security/krb5-tests.py

```diff
@@ -35,22 +35,39 @@ def setUp(self):
         for package in deps:
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
-        url = "https://github.com/krb5/krb5/archive/master.zip"
-        tarball = self.fetch_asset(url, expire='7d')
-        archive.extract(tarball, self.workdir)
-        self.sourcedir = os.path.join(self.workdir, 'krb5-master/src')
-        os.chdir(self.sourcedir)
-        process.run('autoreconf', ignore_status=True)
-        process.run('./configure', ignore_status=True)
-        build.make(self.sourcedir)
+        run_type = self.params.get('type', default='upstream')
+        if run_type == "upstream":
+            default_url = "https://github.com/krb5/krb5/archive/master.zip"
+            url = self.params.get('url', default=default_url)
```

Comment:
make default as None here

Reply:
let default point to url, so test wont fail